vignettes/features.Rmd

```diff
@@ -481,3 +606,118 @@ mtcars |>
     )
   )
 ```
+
+# Putting it all together
+
+```{r}
+iris |> 
+  ggstackplot(
+    # define our shared y axis
+    x = Sepal.Length,
+    # define the stacked x axes
+    y = c(Sepal.Width, Petal.Length, Petal.Width),
+    # We want to altnerate the axes:
+    alternate_axes = TRUE,
+    overlap = 0,
+    # set the base ggplot options
+    template = 
+      # it's a ggplot:
+      ggplot() +
+      # set color aesthetic to vary with species
+      aes(color = Species, fill = Species) +
+      # primary geom will be points
+      #geom_point() +
+      # we want the default stackplot theme
+      theme_stackplot() +
+      # add custom theme modifications
+      theme(
+        # we want each panel to have vertical gridlines
+        panel.grid.major.x = element_line(color = "lightgray", size = 0.2),
+        # and no repeated legends
+        legend.position = "None"
+      ) +
+      # set some labels for the base plot
+      labs(
+        # set the shared axis label:
+        x = "Sepal Length"
+      ),
+    # make panel-specific additions
+    add = list(
+      # for Petal.Length panel:
+      ## >>>> add seems to have issues with labs argument
+      Petal.Length = 
+        geom_col() +
+        labs(y = "Length of the Petal"),
+      Sepal.Width = 
+        geom_point() +
+        labs(y = "Width of the Sepal"),
+      Petal.Width =
+        geom_line() +
+        coord_cartesian(ylim = c(-3, 5)) +
+        labs(y = "Width of the Petal")
+    )
+  )
+```
+
+# Caution
```

Comment:
I've added this caution section to help folks avoid accidentally misrepresenting their data. Perhaps this behavior can generate a warning? "Warning: transforming individual stacked panel axes with `add` is not advised."

Reply:
fixed in version >= 0.2.8